servicemanager.go

```diff
@@ -43,7 +43,10 @@ import (
 	"github.com/aoscloud/aos_servicemanager/monitoring"
 	"github.com/aoscloud/aos_servicemanager/networkmanager"
 	resource "github.com/aoscloud/aos_servicemanager/resourcemanager"
+	"github.com/aoscloud/aos_servicemanager/runner"
```

Comment:
Do we use US english or GB english in commit messages? If US then `initialization`.